views/add_event.hbs

```diff
@@ -2,8 +2,8 @@
   <h3 class="userAddEvent">add event</h3>
 
   <form class="add-event-form" action="/add-event" method="post">
-    <label>Title: </label>
-    <input class="form-control" type="text" name="name" placeholder="title" required>
+    <label>Name: </label>
+    <input class="form-control" type="text" name="name" placeholder="name" required>
```

Comment:
sorry to be picky. I think this needs to have `placeholder='Event name'` or something similar because someone might write their own name. I know this wouldn't have been a problem with `title` but that is the wrong word.